src/main/python/pybuilder_docker_too/__init__.py

```diff
@@ -174,10 +174,11 @@ def _run_docker_container(project, logger, reactor: Reactor):
     logger.debug(f"Running docker with {args}")
     subprocess.Popen(args, stderr=fp_err, stdout=fp)
 
-    _run_pretest_executable(project, logger, reactor)
-
     # give it a bit of time to start up
     time.sleep(3)
+    # run pretest code
+    _run_pretest_executable(project, logger, reactor)
```

Comment:
I get it but this is no longer a pretest executable is it?  Maybe a postinit executable?